dockerutil/dockerfake/client.go

```diff
@@ -163,7 +163,9 @@ func (m MockClient) ContainerExecCreate(ctx context.Context, name string, config
 
 func (m MockClient) ContainerExecInspect(ctx context.Context, id string) (dockertypes.ContainerExecInspect, error) {
 	if m.ContainerExecInspectFn == nil {
-		return dockertypes.ContainerExecInspect{}, nil
+		return dockertypes.ContainerExecInspect{
+			Pid: 1,
```

Comment:
🤔 Does this mean you could accidentally end up killing init if you use this  in a test and run the test as root?

Reply:
umm i think if you set the test up wrong yes probably, should def not make it 1 lol